src/fidesops/ops/service/connectors/email_connector.py

```diff
@@ -77,7 +76,7 @@ def test_connection(self) -> Optional[ConnectionTestStatus]:
                 ],
             )
         except EmailDispatchException as exc:
-            logger.info("Email connector test failed with exception %s", Pii(exc))
+            logger.info("Email connector test failed with exception %s", exc)
```

Comment:
I've updated this, too, to let the email dispatch service handle the PII masking for these exceptions